foundations/tying_it_all_together/conclusion.md

```diff
@@ -1,43 +1,44 @@
 
-Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, building an engineering mindset, working on complex problems and codebases, communicating effectively with other developers, and **much** more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
+Learning how to program is **much** more than just learning a language. It's about learning how to break down your problems into achievable steps, research and solve difficult problems, communicating effectively with other developers, and much more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
 
 Arguments about learning the languages used in your area may sound convincing, but because a _good_ developer can transfer skills easily, what you learn isn't going to massively change outcomes of getting a job. Though, if you cannot shake that fear, do whatever you need, even if that includes researching your area to see what jobs are available locally (don't forget about remote jobs though). A good course is far more important than the language you learn first. 
 
 To help you make a decision, here are some perks of each technology:
 
 #### Ruby and Ruby On Rails: 
-* Opinionated 
-  - You don't have to make decisions on the structure of your projects. This helps when moving into projects built by others.
-  - You will learn things "The Ruby Way", which can help you write clean code in other languages as well.
-* Elegant
-  - Brings a lot of tools to help you develop things in a way that makes your code easy to read.
-  - Often times, solutions in Ruby are much more concise than other languages.
+* Convention over configuration
+  - You don't have to make decisions on the structure of your projects.
+  - Others' Ruby code tend to be similarly structured and thus easy to navigate.
+* Expressive and flexible syntax
+  - Often code can be written in a human-readable style.
+  - Easy to modify existing classes for your usecase.
 * Rapid development
-  - Rails helps you build applications very quickly, this means you can make a greater amount of interesting projects in the same time frame as others.
-* Single paradigm
-  - OOP is the Ruby way, many other languages are primarily OOP based.
+  - Rails is "batteries-included". It comes with solutions to common problems already bundled and ready to use.
+  - Rails can generate project boilerplate with it's scaffolding generators and CLI commands before you even open your editor.
+* Everything is an Object
+  - Community embraces OOP and further leads to consistency across projects.
```

Comment:
"What's OOP"? It's better to write that out fully here.

Reply:
OOP was mentioned in the original version, and I'm thinking if they don't know what "OOP" is they probably won't know what the expanded name means either. That said I'm open to simplifying it a bit, if you think it's necessary.

Reply:
"OOP" just hasn't been introduced to the user (anymore) so for the sake of keeping the text formal or closer to academic, it's better to write it out.

In contrast, the learner has been introduced to what CLI is and should be aware of what CLI stands for, if that makes sense.